core/src/main/java/com/amazon/opendistroforelasticsearch/sql/ast/tree/Head.java

```diff
@@ -38,7 +38,7 @@
 public class Head extends UnresolvedPlan {
```

Comment:
Merge with Limit?

Reply:
The PPL logging will not be able to show correct query since the ppl query data anonymizer is using built AST directly:
```
  @Override
  public String visitHead(Head node, String context) {
    String child = node.getChild().get(0).accept(this, context);
    Integer size = node.getSize();
    return StringUtils.format("%s | head %d", child, size);
  }
```